libpod/container_api.go

```diff
@@ -698,13 +698,26 @@ func (c *Container) WaitForConditionWithInterval(ctx context.Context, waitTimeou
 	resultChan := make(chan waitResult)
 	waitForExit := false
 	wantedStates := make(map[define.ContainerStatus]bool, len(conditions))
+	wantedHealthStates := make(map[string]bool)
 
-	for _, condition := range conditions {
-		switch condition {
-		case define.ContainerStateExited, define.ContainerStateStopped:
-			waitForExit = true
+	for _, rawCondition := range conditions {
```

Comment:
Probably want to sanitize these (lowercase them) before this.

Reply:
That would be a change in behavior.  Podman errors on `--condition=Exited` for instance.

Reply:
I feel like fixing that would be more of a bugfix than anything, but I won't hold up this PR over it.